controllers/servicebinding_controller_test.go

```diff
@@ -818,7 +851,7 @@ var _ = Describe("ServiceBinding controller", func() {
 
 func validateSecretData(secret *v1.Secret, expectedKey string, expectedValue string) {
 	Expect(secret.Data).ToNot(BeNil())
-	Expect(len(secret.Data[expectedKey])).ToNot(BeNil())
+	Expect(secret.Data).To(HaveKey(expectedKey))
```

Comment:
This has nothing to do with the actual PR. But it seemed wrong to us:
`len(...)` will always return an int, therefore the `Expect` would always succeed.